src/components/Contentful/DynamicHeroClassic.vue

```diff
@@ -0,0 +1,335 @@
+<template>
+	<section-with-background-classic
+		:background-content="background"
+		:vertical-padding="verticalPadding"
+	>
+		<template #content>
+			<kv-page-container>
+				<kv-grid
+					class="tw-grid-cols-12 tw-gap-2 tw-items-center"
+					:class="{ 'tw-grid-cols-1 tw-mx-auto': singleColumn }"
```

Comment:
We can now specify if this should be a single column layout instead of a 2 column grid.

Reply:
Can't we just do the 'single column' layout using the centeredRichText type?

Reply:
This component supports expanded image options and specific content areas that the centeredRichText component does not. This component will for all intents and purposes remove the need for the centeredRichText version as it can do that too!